### PR TITLE
Intel AMX accelerated Multicloud GitOps pattern with OpenShift AI 

### PR DESCRIPTION
**Pull request by iirzynska**
In the pattern, on top of Multicloud GitOps pattern, there are two components added: RH OpenShift AI and OpenVINO Toolkit operator. There is also a new folder `scripts` with BERT-Large example, ready to download from Jupyter Notebook (using `wget` command).

### CODE REVIEW

File: charts/all/openvino/templates/ov-notebookinstance.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: intel.com/v1alpha1
+kind: Notebook
+metadata:
```
Comment (mhjacks) [inline charts/all/openvino/templates/ov-notebookinstance.yaml:3]: Would it make sense to make some of these chart parameters, to allow for re-use of the chart?

File: charts/all/openvino/templates/ov-notebookinstance.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  name: v2022.3
+  namespace: redhat-ods-applications
+spec:
+  auto_update_image: false
+  git_ref: main
+  git_uri: 'https://github.com/openvinotoolkit/openvino_notebooks'
```
Comment (mhjacks) [inline charts/all/openvino/templates/ov-notebookinstance.yaml]: If these are parameterized, it definitely makes sense to paramterize git_ref and git_uri

File: charts/all/rhods-operator/templates/rhods-datasciencecluster.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: datasciencecluster.opendatahub.io/v1
```
Comment (mhjacks) [inline charts/all/rhods-operator/templates/rhods-datasciencecluster.yaml:2]: Similar comment about potentially parameterizing - parts of this seems like they could usefully be parameters to enhance re-usability

File: charts/all/openvino/templates/ov-subscription.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (mhjacks) [inline charts/all/openvino/templates/ov-subscription.yaml]: It should not be necessary (and might the gitops operator heartburn) to list the subscription both in the chart and in the subscription section of the clustergroup file. Is this motivated by a problem you had?

File: charts/all/rhods-operator/templates/rhods-subscription.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
```
Comment (mhjacks) [inline charts/all/rhods-operator/templates/rhods-subscription.yaml]: Same comment on this subscription as the previous one.

File: charts/all/openvino/templates/ov-notebookinstance.yaml (hunk 1 above)
Comment (iirzynska) [reply charts/all/openvino/templates/ov-notebookinstance.yaml:3]: Yes, that makes sense. I did some changes here and for rhods-datasciencecluster

File: charts/all/openvino/templates/ov-subscription.yaml (hunk 4 above)
Comment (iirzynska) [reply charts/all/openvino/templates/ov-subscription.yaml]: I had subscription for openvino only in charts and it worked fine. I had problems that openvino was in the _Unknown_ state. I tested it now, works fine, so I moved subscription to values-hub.yaml.

File: charts/all/rhods-operator/templates/rhods-subscription.yaml (hunk 5 above)
Comment (iirzynska) [reply charts/all/rhods-operator/templates/rhods-subscription.yaml]: Same here, subscription was just in charts. But in this case, my main problem was with the OperatorGroup. The managed namespaces were wrong, when I did put RHODS in the regular "pattern way" ( i.e namespace, subscription and project in values-hub.yaml). Adding additional parameters to subscription didn't work and OpenShift AI always ended up in _Unknow_ state (with too many operator groups) or in _Failed_ state (wrong managed namespace) . Thats why I created all of the resources for OpenShift AI in the charts. I didn't know, that more params can be added in the namespace part of values-hub.yaml. We saw example of it in Johnny's repository, I tested it and now it works just fine.

**Review by mhjacks — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**